frontend/src/stylesheets/Home.module.css

```diff
@@ -0,0 +1,238 @@
+/* Pathways To Certification Title */
+
+img {
+  width: 100%;
+}
+
+.title {
+  font: var(--font-title);
+  text-align: center;
+}
+
+.heading {
+  font: var(--font-heading);
+}
+
+.list {
+  margin-left: 2rem;
+}
+
+.olist {
+  margin-left: 1.5rem;
+}
+
+.red {
+  color: var(--color-ccidc-red);
+}
+
+.message {
+  text-align: center;
+}
+
+.item {
+  margin-top: 0 !important;
+}
+
+.textDivider {
+  display: flex;
+  align-items: center;
+  --text-divider-gap: 1rem;
+}
+
+.textDivider::after {
+  content: "";
+  height: 1px;
+  background-color: #d8d8d8;
+  flex-grow: 1;
+  margin-top: 3em;
+  margin-left: var(--text-divider-gap);
+}
+
+.centeredContainer {
+  display: flex;
+  justify-content: center;
+  padding-bottom: 3em;
+}
+
+.textDivider2 {
+  display: flex;
+  align-items: center;
+  --text-divider-gap: 1rem;
+}
+
+.textDivider2::before {
+  content: "";
+  height: 1px;
+  background-color: #d8d8d8;
+  flex-grow: 1;
+  margin-right: var(--text-divider-gap);
+}
+.textDivider2::after {
+  content: "";
+  height: 1px;
+  background-color: #d8d8d8;
+  flex-grow: 1;
+  margin-left: var(--text-divider-gap);
+}
+
+.imgs {
+  display: flex;
+  align-items: center;
+  justify-content: space-evenly;
+  padding: 1em;
+}
+
+.imgCaption {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 18rem;
+}
+
+.cert {
+  max-width: 194px;
+}
+
+.caption {
+  text-align: center;
+}
+
+/* Pathways */
+
+h3 {
+  font-size: 24px;
+}
+
+.pathwaysContainer {
+  display: grid;
+  grid-template-columns: repeat(2, 1fr); /* 2 columns */
+  grid-template-rows: repeat(2, auto); /* 2 rows */
+  gap: 2rem; /* gap between grid items */
+}
+
+@media screen and (max-width: 1385px) {
+  .pathwaysContainer {
+    display: flex;
+    flex-wrap: wrap;
+    flex-direction: column;
+    row-gap: 2rem;
+    align-items: center;
+  }
+}
+
+.pathwayContainer {
+  width: 38rem;
+  height: 400px;
+  border-radius: 10px;
+  border: 1px solid #d8d8d8;
+}
+
+.pathName {
+  margin-top: 1.7rem;
+  margin-left: 1.5rem;
+}
+
+.pathwayText {
+  width: auto;
+  height: 5.5rem;
+  margin: 0.875rem 1.5rem;
+  line-height: 1.5em;
+}
+
+/* FAQ's Title, Contianer, and Components */
+
+.FaqsContainer {
+  display: flex;
+  flex-wrap: wrap;
+  flex-direction: column;
+  row-gap: 0.8rem;
+}
+
+.faqBar {
+  display: flex;
+  flex-direction: row;
+  column-gap: 0.7rem;
+  align-items: center;
+}
+
+.faqTitle {
+  width: 1248px;
+  height: 65px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  --text-divider-gap: 1rem;
+  font-size: 28px;
+}
+
+.infoTitle {
+  width: 1248px;
+  height: 56px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.infoTitle h2 {
+  align-self: center;
+  margin-top: 0em;
+}
+
+.line {
+  content: "";
+  height: 1px;
+  background-color: #d8d8d8;
+}
+
+.expand {
+  display: flex;
+  width: 145px;
+  height: hug(40px);
```

Comment:
hug does not exist in the css. Which I think is causing the problem that you try to resolve using expandAll's css rules. Generally if you want to use css from the figma file use the figma developer mode and look at it from there. Though now that i might have extra excessive rules that might not be necessary.